### DISCUSSION TITLE
Turkish Language Translation

### DISCUSSION THREAD

**Discussion by sonmiralay** [lostindark/DriverStoreExplorer#170] [Show and tell]
I want help translate for Turkish Language. How can I help?

**Comment by SonicWim** [comment]
I also'd like to help but with Dutch translations.

**Comment by lostindark** [comment]
You can use the English language as a template and create your language translation:
https://github.com/lostindark/DriverStoreExplorer/blob/master/Rapr/Lang/Language.resx

You can create a pull request directly with the required change. Here is an example of adding a translation language:
https://github.com/lostindark/DriverStoreExplorer/commit/b94cee8b45c6d5562118a0d45b9fffe9c520bda6

Or send me the translated file.
